src/test/java/ay2122s1_cs2103t_w16_2/btbb/testutil/GenericDummy.java

```diff
@@ -0,0 +1,30 @@
+package ay2122s1_cs2103t_w16_2.btbb.testutil;
+
+import ay2122s1_cs2103t_w16_2.btbb.model.shared.GenericString;
+
+/**
+ * Dummy utility class to test generic classes.
+ */
+public class GenericDummy {
```

Comment:
lol why is this called `GenericDummy` anyway it's only used to test predicates right? Will it be used for anything else? Is it actually used to test generic classes? What about `PredicateUtil` if it won't be used for anything else?